ext/ChainRulesCoreExt.jl

```diff
@@ -2,12 +2,15 @@
 
 module ChainRulesCoreExt
 
-using CUDA: CuArray
+using CUDA: CuArray, CUDA
 
 isdefined(Base, :get_extension) ? (import ChainRulesCore) : (import ..ChainRulesCore)
 
 ## support ChainRulesCore inplaceability
 
 ChainRulesCore.is_inplaceable_destination(::CuArray) = true
 
+# allow usage of rand with Zygote
+ChainRulesCore.@non_differentiable CUDA.randn(::Any...)
```

Comment:
`rand` too?
```suggestion
ChainRulesCore.@non_differentiable CUDA.rand(::Any...)
ChainRulesCore.@non_differentiable CUDA.randn(::Any...)
```
Tab completion says there are a few more, but not marked public, so IDK:
```
julia> CUDA.rand
rand              rand_logn
rand_logn!        rand_poisson
rand_poisson!     randexp_unlikely
randn             randn_unlikely
```